WordPress/Classes/Services/EditorSettingsService.swift

```diff
@@ -19,11 +31,13 @@ import Foundation
         service.getEditorSettings(siteID, success: { (settings) in
             do {
                 try self.saveRemoteEditorSettings(settings, on: blog)
+                ContextManager.sharedInstance().save(self.managedObjectContext)
+                success()
+            } catch EditorSettingsServiceError.mobileEditorNotSet {
+                self.migrateLocalSettingToRemote(for: blog, success: success, failure: failure)
```

Comment:
As mentioned in Slack, we bail early from `saveRemoteEditorSettings`, and we never set the `webEditor` value in this branch, until the `syncEditorSettings` is called the next time

Reply:
Pushed https://github.com/wordpress-mobile/WordPress-iOS/pull/12189/commits/5fef12b96b156fa4fc0103c08e430a4bd9ec7cff